.circleci/config.yml

```diff
@@ -229,15 +214,15 @@ references:
     paths:
       - packages
       - packages-cache-restored
-  desktop-restore-webpack-css-loader-cache: &desktop-restore-webpack-css-loader-cache
-    name: Restore webpack cache-loader cache
+  desktop-restore-calypso-build-cache: &desktop-restore-calypso-build-cache
+    name: Restore Calypso build cache
     keys:
-      - v6-desktop-webpack-css-loader-{{ .Environment.CIRCLE_JOB }}-{{ arch }}-{{ checksum "webpack-css-loader-hash" }}
-  desktop-save-webpack-css-loader-cache: &desktop-save-webpack-css-loader-cache
-    name: Save webpack cache-loader cache
-    key: v6-desktop-webpack-css-loader-{{ .Environment.CIRCLE_JOB }}-{{ arch }}-{{ checksum "webpack-css-loader-hash" }}
+      - v7-desktop-calypso-build-{{ .Environment.CIRCLE_JOB }}-{{ arch }}
+  desktop-save-calypso-build-cache: &desktop-save-calypso-build-cache
+    name: Save Calypso build cache
+    key: v7-desktop-calypso-build-{{ .Environment.CIRCLE_JOB }}-{{ arch }}
```

Comment:
Changes look good, but one more quick question:

This cache key is limited to `CIRCLE_JOB` and `arch`, which won't change between CI jobs/branches/SHAs. It doesn't seem that the cache will ever be invalidated, unless I'm missing something?

In contrast, the key in for Calypso is `v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v0-build-{{ .Branch }}-{{ .Revision }}`, which at the very least has the revision. Should we use a similar setup here as well?

Reply:
Ah right, forgot that caches are immutable in CircleCI. Added `.Branch` and `.Revision` as part of the key like Calypso.
Good catch!